crates/nu-command/tests/commands/split_by.rs

```diff
@@ -0,0 +1,61 @@
+use nu_test_support::fs::Stub::EmptyFile;
```

Comment:
I think this file is supposed to have been removed. I thought it was removed recently in a PR along with the split-by command.

Reply:
shoot, i messed up the merge. thanks for catching this!